policy/scan/local_scanner.go

```diff
@@ -381,7 +381,7 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			assets[i].asset.Url = platformAssetMapping[platformMrn].Url
 		}
 	} else {
-		// ensure we have non-empty asset MRNs
+		// ensure we have no non-empty asset MRNs
```

Comment:
i think the comment was ok as it was...we want to make sure the mrns are not empty, adding no there makes it seem like we want empty mrns

Reply:
I added it because I thought that's the intention to **not** have non-empty mrns - maybe here it shines through that english is not my native language but to me it sounded wrong without the no

Reply:
Changed it back